src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -30,7 +30,7 @@ abstract class Address {
   @SequenceGenerator(name = "ADDRESS_ID", sequenceName = "ADDRESS_ID", allocationSize = 1)
   @GeneratedValue(generator = "ADDRESS_ID")
   @Column(name = "ADDRESS_ID", nullable = false)
-  val addressId: Long? = null
+  val addressId: Long = 0
```

Comment:
this avoids the nullable id issue

Reply:
Ah yes, I remember now;  zero is treated like it is not set in JPA